packages/tooltip/src/tooltipUtils.tsx

```diff
@@ -0,0 +1,105 @@
+import { css } from '@leafygreen-ui/emotion';
+
+export function trianglePosition(
+  alignment: string,
+  justification: string,
+  triggerRect: DOMRect | ClientRect | null,
+) {
+  if (!alignment || !justification || !triggerRect) {
+    return '';
+  }
+
+  const borderOffset = 'calc(100% - 1px)';
+
+  const containerSize = 15;
+  const notchSize = 8;
+  const containerOffsetX = (triggerRect.width - containerSize) / 2;
+  const containerOffsetY = (triggerRect.height - containerSize) / 2;
+  const notchOverlap = -notchSize / 2;
+  const containerStyleObj: any = {};
+  const notchStyleObj: any = {};
+
+  switch (alignment) {
+    case 'top':
+    case 'bottom':
+      notchStyleObj.left = 0;
+      notchStyleObj.right = 0;
+
+      if (alignment === 'top') {
+        containerStyleObj.top = `${borderOffset}`;
+        notchStyleObj.top = `${notchOverlap}px`;
+      } else {
+        containerStyleObj.bottom = `${borderOffset}`;
+        notchStyleObj.bottom = `${notchOverlap}px`;
+      }
+
+      switch (justification) {
+        case 'left':
+          containerStyleObj.left = `${containerOffsetX}px`;
+          break;
+
+        case 'center-horizontal':
+          containerStyleObj.left = '0px';
+          containerStyleObj.right = '0px';
+          containerStyleObj.margin = 'auto';
```

Comment:
We should be able to set this style more outside a conditional for this element.

Reply:
Not sure what you mean sorry

Reply:
We can probably unconditionally set margin to `auto`.

Reply:
👍